src/core/qgsmaptopixel.h

```diff
@@ -39,6 +39,13 @@ class CORE_EXPORT QgsMapToPixel
 {
   public:
 
+    /**
+     * Constructor for an invalid QgsMapToPixel.
+     *
+     * A manual call to setParameters() is required to initialize the object.
```

Comment:
```suggestion
     * A manual call to setParameters() is required to initialize the object.
     * \since QGIS 3.22
```

Reply:
It's not a new method, it's been around forever 😂